WooCommerce/Classes/ViewModels/Order Details/OrderDetailsResultsControllers.swift

```diff
@@ -34,6 +34,15 @@ final class OrderDetailsResultsControllers {
     ///
     private lazy var productVariationResultsController: ResultsController<StorageProductVariation> = getProductVariationResultsController()
 
+    /// Fee lines Results Controller.
+    ///
+    private lazy var feeLinesResultsController: ResultsController<StorageOrderFeeLine> = {
+        let predicate = NSPredicate(format: "order.orderID == %ld", order.orderID)
+        let descriptor = NSSortDescriptor(key: "name", ascending: true)
```

Comment:
I did see the order of multiple custom amount lines jumping around a bit as I tested. Perhaps this sort order isn't needed, or we need it somewhere else as well? Probably not a big thing, but I bet we'll have some users who use 3+ custom amounts in an order rather than use products, and it might be a bit weird for them.

Sorting on ID might be more appropriate here, since the rest of the order items aren't going to be alphabetised by name.

Reply:
> Sorting on ID might be more appropriate here, since the rest of the order items aren't going to be alphabetised by name

That's a good point, thanks for suggesting that! Changed in ef7e151e005f3616a3dc6d179b3598f53acd4e65